.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: edvinas108/meme-grid:latest
```

Comment:
I'd suggest to use one of the CircleCI images, e.g. `circleci/openjdk:8...` or a plain `openjdk` image. 

Reasoning for this:
* Current `Dockerfile` is not that well built - it doesn't take Java version into account.
* It's not optimized for running tests - it includes the resulting `.jar` file. 
* In the future it might be updated to include specific run configs which are not necessary for running tests.

Reply:
Change completed to use the circleci/openjdk:8 image